### DISCUSSION TITLE
[New Feature] MindControl affect passengers

### DISCUSSION THREAD

**Discussion by Cpj636** [Phobos-developers/Phobos#897] [Ideas and feedback]
<pre>
1.Just like the picture below shows, units being MCed can be MCed again.
![image](https://user-images.githubusercontent.com/80351894/113557139-1d5a8400-9630-11eb-8135-b6527149e0f5.png)

2.MindControlRangeLimit seems not to work, just like the picture below shows
![image](https://user-images.githubusercontent.com/80351894/113558121-ac1bd080-9631-11eb-86f7-d4fbe0efdb13.png)

3.Some other Suggestions 
About Feature TransactMoney
Is it possible to add a tag like "TransactMoneyDisplay", just like "Bounty.Display","ProduceCashDisplay". Then, we do not need to draw anims for each unit with such a feature.

About Mind Control
When a unit with passengers in it, after being MCed, its passengers will not be affected, which is stupid especially for opentopped units. Moreover, opentopped units after being MCed, its passengers can fire both at the same target as the vehicle as well as fire the owners' units.
Therefore, whether it is possible to add some features like:
"MindControl.ReleasePassengers" or "MindControl.IncludePassengers" or "MindControl.KillPassengers"
</pre>

***
***
***

```ini
MindControl.ReleasePassengers=boolean
MindControl.IncludePassengers=boolean
MindControl.KillPassengers=boolean
```

**Comment by Thrifinesma** [comment]
The first two have been fixed in newer Phobos build#9, you can redownload it

**Comment by Cpj636** [comment]
> The first two have been fixed in newer Phobos build#9, you can redownload it

Thanks for your team. This project makes the game much more interesting.

**Comment by Cpj636** [comment]
> The first two have been fixed in newer Phobos build#9, you can redownload it

Another issue to report, weapons without damage(damage=1,warhead versus 5%) such as pure emp weapons cannot target at a unit with shield. Even force it to attack, still no responses until the shield is broken by others.

**Comment by Thrifinesma** [comment]
> > The first two have been fixed in newer Phobos build#9, you can redownload it
> 
> Another issue to report, weapons without damage(damage=1,warhead versus 5%) such as pure emp weapons cannot target at a unit with shield. Even force it to attack, still no responses until the shield is broken by others.

Unfortunately, we now can not judge whether a weapon is able to target shield (because ArmorType-related is Ares' feature), so the Shields currently can only be targeted when Damage=0 or Damage*Versus≠0. Might change in the future. Thanks for your report

**Comment by secsome** [comment]
```ini
MindControl.ReleasePassengers
MindControl.IncludePassengers
MindControl.KillPassengers
```
Should be split into a separate issue and close this one.
